src/m365/tenant/commands/report/report-settings-set.ts

```diff
@@ -0,0 +1,102 @@
+import { Logger } from '../../../../cli/Logger.js';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import GraphCommand from '../../../base/GraphCommand.js';
+import commands from '../../commands.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
```

Comment:
For options and validation you are currently using the old way, we've recently moved to zod, which is a tool with which we can more easily parse options and validate the schema. Would you mind switching the code to use zod? You can find an example here: 
`pnp\cli-microsoft365\src\m365\entra\commands\roledefinition\roledefinition-get.ts`